Zotero/Scenes/General/ViewModels/TagPickerActionHandler.swift

```diff
@@ -92,9 +92,10 @@ struct TagPickerActionHandler: ViewModelActionHandler {
         do {
             let request = ReadTagPickerTagsDbRequest(libraryId: viewModel.state.libraryId)
             let results = try self.dbStorage.perform(request: request, on: .main)
-            let colored = results.filter("color != \"\"").sorted(byKeyPath: "name")
-            let others = results.filter("color = \"\"").sorted(byKeyPath: "name")
-            let tags = Array(colored.map(Tag.init)) + Array(others.map(Tag.init))
+            let colored = results.filter("color != \"\"").sorted(byKeyPath: "order")
```

Comment:
Why did the sorting changed here?

Reply:
I matched desktop behavior. I didn't notice the difference before.